nautobot/dcim/api/serializers.py

```diff
@@ -200,6 +200,21 @@ class Meta:
         # https://www.django-rest-framework.org/api-guide/validators/#optional-fields
         validators = []
 
+        detail_view_config = [
+            {
+                "Location": {
+                    "fields": ["name", "location_type", "tenant", "description", "facility", "asn", "time_zone"]
+                },
+                "Contact Info": {
+                    "fields": ["physical_address", "shipping_address", "contact_name", "contact_phone", "contact_email"]
+                },
```

Comment:
Missing `latitude` and `longitude` fields. Is there anything we can add in the detail_view_config processing to automatically detect if fields are missing from the config?

Reply:
What of the user do not want to show all model fields?

Reply:
Let's discuss this in standup tomorrow then. I think right now it's too easy to inadvertently omit fields that should be present.